charts/eac/values.yaml

```diff
@@ -71,6 +75,8 @@ fuse:
   #      memory: "4G"
 
 master:
+  labels: {}
```

Comment:
Please also update chart version and changelog.

Reply:
Done